task1.js

```diff
@@ -1,28 +1,23 @@
 document.addEventListener("DOMContentLoaded", function() {
   var btn1 = document.getElementById("btn1");
   var btn2 = document.getElementById("btn2");
-  var counter = 0;
-  var newButton = '<button type="button" class="btn btn-success">NowyKliker</button>'
+
   var p1 = document.getElementById("p1");
   var p2 = document.getElementById("p2");
 
+  var counter = 0;
+  var newButton = '<button type="button" class="btn btn-success">NowyKliker</button>'
 
-  function clicks(event){
+  function clicks(event) {
     counter += 1;
     p1.innerHTML = "click " + counter;
   }
 
-  var newButtonClicks = $("#p2");
-  newButtonClicks.on('click','button', function(){
-    counter += 1;
-    p1.innerHTML = "click " + counter;
-  });
-
-  function add_buton(event){
+  function addButton(event) {
     p2.innerHTML += newButton;
   }
 
-  btn1.addEventListener("click" , clicks);
-  btn2.addEventListener("click" , add_buton);
-
-})
+  $(p2).on('click', 'button', clicks);
```

Comment:
1. Patrz, tutaj możesz użyć `p2` - jak zrobisz `$(<zwykły element>)`, to masz obiekt jQuery odnoszący się do tego elementu HTML. Tak jest czyściej.

2. Dwa razy miałeś zdefiniowaną funkcję, która robi dokładnie to samo - wywaliłem tę duplikację.